app/views/gettingStarted.scala.html

```diff
@@ -1,40 +1,109 @@
-@(title: String = "Try Play")(implicit requestHeader: RequestHeader)
+@(examples: PlayExamples)(implicit requestHeader: RequestHeader)
 
-@main(title, "getting-started") {
+
+@renderProject(project: ExampleProject) = {
+    <tr>
+        <td><a href="@project.downloadUrl" target="_blank">@project.displayName</a></td>
```

Comment:
@rstento we get data for sample projects using Tech Hub API. The only URLs available there are for download and Github. To avoid hardcoding the links, I've decided to keep the current way of listing the starter projects.

Reply:
I think that's OK for now, but we can do some usability improvements later. It will help when Tech Hub allows us to filter for Java and Scala as mentioned in my review of the doc changes.